addons/website_blog/migrations/13.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,71 @@
+---Models in module 'website_blog'---
+---Fields in module 'website_blog'---
+website_blog / blog.blog                / content (html)                : NEW
+website_blog / blog.blog                / cover_properties (text)       : NEW hasdefault
+# NOTHING TO DO
```

Comment:
This is not nothing to do, but we should adapt previous properties for the cover to this new field.